Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rsrchboy/perl-v5.16
+MAINTAINER Xinjiang Shao <shaoxinjiang@gmail.com>
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    git \
+    libssl-dev \
+    libmpfr-dev \
+    nodejs \
+    npm \
+    nodejs-legacy \
+    && rm -fr /var/lib/apt/lists/*
+RUN npm install -g handlebars@1.3.0 uglifyjs
```

Comment:
The thing is if we don't install npm, nodejs, we can not have handlerbars; if we don't have handlerbars, when you run dzil listdeps, it will trigger an error message. So we might as well install all the required dependencies to get rid of the errors.